docs/UserGuide.md

```diff
@@ -226,39 +242,35 @@ Examples:
 * `find university/NUS` followed by `export` exports the result of the `find` command to a JSON file.
 
 
-### 3.9 Deleting an applicant: `delete`
+### 3.10 Check-out a new or existing list: `checkout`
 
-Deletes the specified applicant from InternConnect.
+Switches between different lists in InternConnect.
 
-Format: `delete INDEX`
+Format: `checkout file_name`
```

Comment:
Please use CAPITAL letters here

Reply:
Line changed to `Format: `checkout FILE_NAME``